gapic/templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2

```diff
@@ -1293,7 +1293,7 @@ def test_{{ method.name|snake_case }}_rest_flattened(transport: str = 'rest'):
         assert len(req.mock_calls) == 1
         _, args, _ = req.mock_calls[0]
         {% with uri = method.http_options[0].uri %}
-        assert path_template.validate("https://{{ service.host }}{{ uri }}", args[1])
+        assert path_template.validate("https://%s{{ uri }}" % client.transport._host, args[1])
```

Comment:
Do we not have a "golden" example for a REST API?

Reply:
No we do not, but I think we should.

Reply:
No we do not, but I think it is a good idea to have one.